app/Config/bootstrap.php

```diff
@@ -94,21 +95,21 @@
  * ));
  */
 Configure::write('Dispatcher.filters', array(
-	'AssetDispatcher',
-	'CacheDispatcher'
+    'AssetDispatcher',
```

Comment:
__[staging][code_sniffer]__
* Array value not aligned correctly; expected 40 spaces but found 4